flycheck.el

```diff
@@ -7439,6 +7500,17 @@ See URL `https://github.com/eslint/eslint'."
             (option-list "--rulesdir" flycheck-eslint-rules-directories)
             "--stdin" "--stdin-filename" source-original)
   :standard-input t
+  :enabled (lambda (checker)
+             (let* ((command (concat (flycheck-find-checker-executable checker)
+                                     (if flycheck-eslintrc
+                                         (concat " --config " flycheck-eslintrc)
+                                       "")
+                                     " --print-config "
+                                     (flycheck-compute-working-directory checker)
+                                     "."))
+                    (exitcode (call-process-shell-command command nil nil nil)))
+               (cond ((eq exitcode 0) nil)
+                     (t "Configuration not found"))))
```

Comment:
Thanks, that's nice, but would you mind to remove the changes from the eslint checker again?  Let's keep this PR limited to `:enabled`, and make second one for the change to eslint?  That's just easier for me to follow, and I believe it's better to keep pull requests confined to just one change.